airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -124,7 +124,10 @@ class DataflowConfiguration:
         WaitForRun = wait until job finished and the run job.
         Supported only by:
         :py:class:`~airflow.providers.apache.beam.operators.beam.BeamRunJavaPipelineOperator`
-    :param service_account: Run the job as a specific service account, instead of the default GCE robot.
+    :param service_account: Run the Java job as a specific service account, instead of the default Compute Engine service account.
+        See: https://cloud.google.com/dataflow/docs/reference/pipeline-options#java
+    :param service_account_email: Run the Python or Go job as a specific service account for Python or Go, instead of the default Compute Engine service account.
+        See: https://cloud.google.com/dataflow/docs/reference/pipeline-options#python or https://cloud.google.com/dataflow/docs/reference/pipeline-options#go
```

Comment:
I just wonder if it doesn't make sense to put these into a single parameter rather than two separate ones? The documentation is quite clear, so it may not be a big deal - but ideally users only have to care about one parameter?

I understand that complicates the logic downstream, but maybe it's better to complicate downstream than enable users to misconfigure/be confused?

also, sorry about the delay ^^'

Reply:
How well does the API handle unknown keys? If unknown keys are simply ignored, we could probably just always set both keys. Otherwise the implementation might be pretty complicated and not worth it.

Reply:
once again my apologies for missing this - I am not sure about how the API deals with this. If not, I am happy to move forward with the way it is...